portfolio/src/main/webapp/image_script.js

```diff
@@ -0,0 +1,39 @@
+/** A program that handles the image gallary */
+const numImages = 8;    // The number of images
+let imageIdx = 0;       // The index of current image being shown
+
+/** Displays the previous image */
+function showPrevImage() {
+  // Decrease the index by 1, round up to the last image if the index become -1
+  imageIdx = (imageIdx - 1 + numImages) % numImages;
+  showImageWithCurrentIdx();
+}
+
+/** Displays the next image */
+function showNextImage() {
+  /* Increase the index by 1, round up to the first image if the index is
+     larger than the number of images */
+  imageIdx = (imageIdx + 1) % numImages;
+  showImageWithCurrentIdx();
+}
+
+/** Displays the image with the current index */
+function showImageWithCurrentIdx() {
+  // Change the header to show the current index of the image being shown
+  const headerElement = document.getElementById('petsImage')
+                        .getElementsByTagName('h1')[0];
```

Comment:
per https://screenshot.googleplex.com/nMLdzbk8fES, consider refactoring this line break?

Reply:
Done!